python/requirements.txt

```diff
@@ -1,115 +1,24 @@
 appdirs==1.4.4
-argh==0.26.2
-attrs==20.2.0
-aws-sam-translator==1.28.1
 aws-xray-sdk==2.6.0
-batchglm==0.7.4
 black==20.8b1
-boto==2.49.0
 boto3==1.16.9
 botocore==1.19.9
-certifi==2020.6.20
-cffi==1.14.3
-cfn-lint==0.39.0
-chardet==3.0.4
-click==7.1.2
 coverage==5.3
-cycler==0.10.0
-dask==2.30.0
-decorator==4.4.2
-diffxpy==0.7.4
 docker==4.3.1
-ecdsa==0.14.1
 flake8==3.8.4
 future==0.18.2
-get-version==2.1
-h5py==2.10.0
-idna==2.10
-importlib-metadata==2.0.0
-iniconfig==1.1.1
-Jinja2==2.11.3
-jmespath==0.10.0
-joblib==0.17.0
-jsondiff==1.2.0
-jsonpatch==1.26
-jsonpickle==1.4.1
-jsonpointer==2.0
-jsonschema==3.2.0
-junit-xml==1.9
-kiwisolver==1.3.1
-legacy-api-wrap==1.2
-leidenalg==0.8.2
-llvmlite==0.34.0
-loompy==3.0.6
-louvain==0.7.0
-MarkupSafe==1.1.1
-matplotlib==3.3.2
-mccabe==0.6.1
 mock==4.0.2
-more-itertools==8.6.0
-moto==1.3.16
-mypy-extensions==0.4.3
 natsort==7.0.1
-networkx==2.5
-numba==0.51.2
-numexpr==2.7.1
 numpy==1.19.3
-numpy-groupies==0.9.13
-packaging==20.4
 pandas==1.1.4
-pandasql==0.7.3
-pathspec==0.8.0
-pathtools==0.1.2
-patsy==0.5.1
-Pillow==8.3.2
-pluggy==0.13.1
-py==1.10.0
-pyasn1==0.4.8
-pycodestyle==2.6.0
-pycparser==2.20
-pyflakes==2.2.0
-pyparsing==2.4.7
-pyrsistent==0.17.3
 pytest==6.1.2
-pytest-cov==2.10.1
```

Comment:
[pytest-cov](https://pypi.org/project/pytest-cov/) is needed to produce test coverage. I don't think it's used in our CI for code coverage, but it certainly can be used in local.